js/components/interface/graph-visualization/forces/center.js

```diff
@@ -0,0 +1,63 @@
+export default function (x, y, z) {
```

Comment:
What about a description on the top about what this file is defining? Probably giving a name to the function would also help

Reply:
I mean just some comment on the top of the file to understand why they are there